client/Components/GetInTouchForm.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+
+const GetInTouchForm = ( { handleFormChange }) => {
+
+  const [isHidden, toggleHidden] = useState(true);
+
+  const handleFormFieldFocus = (e) => {
+
+    const field = e.target.parentElement;
+    field.classList.add('focus');
+
+  }
+
+  const handleFieldBlur = (e) => {
+
+    const field = e.target.parentElement;
+
+    //can eventually add other helper functions here to do some *basic* front end validation
+    if (!e.target.value) {
+
+      if (field.classList.contains('complete')) field.classList.remove('complete');
+
+      field.classList.add('error');
+      toggleHidden(false);
```

Comment:
known bug: this toggles on same error message no matter what field it was triggered by